pkg/clusteragent/clusterchecks/dispatcher_main.go

```diff
@@ -41,18 +43,16 @@ func (d *dispatcher) Unschedule(configs []integration.Config) {
 	}
 }
 
-// Stop implements the scheduler.Scheduler interface
-// no-op for now
-func (d *dispatcher) Stop() {
-}
-
 // add stores and delegates a given configuration
 func (d *dispatcher) add(config integration.Config) {
 	if !config.ClusterCheck {
 		return // Ignore non cluster-check configs
 	}
 	log.Debugf("dispatching configuration %s:%s", config.Name, config.Digest())
-	d.store.addConfig(config)
+
+	// TODO: add dispatching logic
+	hostname, _ := util.GetHostname()
```

Comment:
If I get this right, currently all cluster checks are dispatched to the same host as the cluster-agent leader?

Reply:
Yes, this PR only brings in the piping with a placeholder logic, so the next PR can focus on the dispatching logic and be easier to review